ReactViewControl/ViewModuleContainer.cs

```diff
@@ -80,18 +83,28 @@ public abstract class ViewModuleContainer : IViewModule {
             var entriesFilePath = VirtualPathUtility.GetDirectory(MainJsSource) + Path.GetFileNameWithoutExtension(MainJsSource) + extension;
             var resource = entriesFilePath.Split(new[] { ResourceUrl.PathSeparator }, StringSplitOptions.RemoveEmptyEntries);
 
-            var stream = ResourcesManager.TryGetResourceWithFullPath(resource.First(), resource);
-            if (stream != null) {
-                using (var reader = new StreamReader(stream)) {
-                    var allEntries = reader.ReadToEnd();
-                    if (allEntries != null && allEntries != string.Empty) {
-                        return allEntries.Split(new[] { "\n" }, StringSplitOptions.RemoveEmptyEntries);
+            using (var stream = GetResourceStream(resource)) {
+                if (stream != null) {
+                    using (var reader = new StreamReader(stream)) {
+                        var allEntries = reader.ReadToEnd();
+                        if (allEntries != null && allEntries != string.Empty) {
+                            return allEntries.Split(new[] { "\n" }, StringSplitOptions.RemoveEmptyEntries);
+                        }
                     }
                 }
+
             }
             return new string[0];
         }
 
+        private Stream GetResourceStream(string[] resource) {
+            isHotReloadEnabled = childViewHost == null ? false : childViewHost.IsHotReloadEnabled;
```

Comment:
this field useless... always being assigned, delete

Reply:
field removed but I need to check if the childView is null